INFO     HTTP_CLIENT:event_hooks.py:28 Make POST request to http://localhost:8000/api/v1/users
INFO     HTTP_CLIENT:event_hooks.py:38 Got response 200 OK from http://localhost:8000/api/v1/users
INFO     HTTP_CLIENT:event_hooks.py:28 Make POST request to http://localhost:8000/api/v1/authentication/login
INFO     HTTP_CLIENT:event_hooks.py:38 Got response 200 OK from http://localhost:8000/api/v1/authentication/login
INFO     HTTP_CLIENT:event_hooks.py:28 Make POST request to http://localhost:8000/api/v1/exercises
INFO     HTTP_CLIENT:event_hooks.py:38 Got response 200 OK from http://localhost:8000/api/v1/exercises
INFO     HTTP_CLIENT:event_hooks.py:28 Make DELETE request to http://localhost:8000/api/v1/exercises/0e229efa-5e42-4921-9911-15af86a5e39e
INFO     HTTP_CLIENT:event_hooks.py:38 Got response 200 OK from http://localhost:8000/api/v1/exercises/0e229efa-5e42-4921-9911-15af86a5e39e
INFO     BASE_ASSERTIONS:base.py:19 Check that response status code equals to 200
INFO     HTTP_CLIENT:event_hooks.py:28 Make GET request to http://localhost:8000/api/v1/exercises/0e229efa-5e42-4921-9911-15af86a5e39e
INFO     HTTP_CLIENT:event_hooks.py:38 Got response 404 Not Found from http://localhost:8000/api/v1/exercises/0e229efa-5e42-4921-9911-15af86a5e39e
INFO     BASE_ASSERTIONS:base.py:19 Check that response status code equals to 404
INFO     BASE_ASSERTIONS:exercises.py:106 Check that "details='Exercise not found'" equals to details='Exercise not found'
INFO     BASE_ASSERTIONS:base.py:38 Check that "details" equals to Exercise not found